src/aws_metadata.app.src

```diff
@@ -5,7 +5,8 @@
   {mod, {aws_metadata_app, []}},
   {applications,
    [kernel,
-    stdlib
+    stdlib,
+    aws
```

Comment:
The `aws` application doesn't have any state... so it shouldn't need
to be started.  What problems are you seeing that this solves?